pkg/resources/otel_conf_gen/pipeline/components/common.go

```diff
@@ -106,6 +110,46 @@ func (r *ResourceRelations) GetTenantByName(tenantName string) (*v1alpha1.Tenant
 	return nil, fmt.Errorf("tenant %s not found", tenantName)
 }
 
+// QueryOutputSecret retrieves the secret associated with the output's authentication configuration
+// It returns an error if the output is nil or if the secret cannot be found.
+func QueryOutputSecret(ctx context.Context, client client.Client, output *v1alpha1.Output) error {
+	if output == nil {
+		return errors.New("output cannot be nil")
+	}
+
+	_, err := QueryOutputSecretWithData(ctx, client, output)
+	return err
+}
```

Comment:
I don't think this function adds much value in its current form.
Since `QueryOutputSecretWithData` already does the heavy lifting and returns both the secret and an error, callers can simply discard the secret if they don't need it. This wrapper doesn’t provide additional abstraction or simplification.
What do you think?